src/settings/template.ts

```diff
@@ -163,6 +164,7 @@ export const renderArticleContnet = async (
   dateSavedFormat: string,
   isSingleFile: boolean,
   frontMatterVariables: string[],
```

Comment:
Does Obsidian handle converting these into an array for us? It looks like it works well, i tested out some scenarios where i add extra whitespace and it was trimmed properly.

Reply:
Oh, I convert and trim those in the `onChange` callback